Gemfile.lock

```diff
@@ -331,7 +329,7 @@ GEM
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webdrivers (4.5.0)
+    webdrivers (4.6.0)
```

Comment:
### [webdrivers](https://github.com/titusfortner/webdrivers)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.5.0 → 4.6.0

[_[change-log](https://github.com/titusfortner/webdrivers/blob/master/CHANGELOG.md), [source-code](https://github.com/titusfortner/webdrivers/tree/v4.6.0)_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/titusfortner/webdrivers/compare/v4.5.0...v4.6.0).

These are the individual commits:
- (abac425) [Stop using Appveyor and TravisCI.](https://github.com/titusfortner/webdrivers/commit/abac425923cf5cdf9794db0541be368020ac8bfd)
- (6eb6027) [Update browser & OS specific notes in README](https://github.com/titusfortner/webdrivers/commit/6eb6027f6258a9fd3a60a79c122ac1fbf7cc6a37)
- (850e910) [rename Edge classes to support Selenium 4 beta](https://github.com/titusfortner/webdrivers/commit/850e9107adaf4d948c430c228773298ef180eca0)
- (0b608d1) [bump to v4.6.0](https://github.com/titusfortner/webdrivers/commit/0b608d10d55112ae0b3cf41ed5951c2f6a2ac9e4)


</details>